Evaluation-2/src/expression-test.js

```diff
@@ -0,0 +1,127 @@
+import { createStack } from "./stack.js"
+export function expressionTest(){
+let stack=createStack()
+return{
+    checkExpr(expr){
+        let str=expr;
+        let expArr=[]
+        for(let i=0;i<str.length;i++){
+            if(str[i]=='1'||str[i]=='2'||str[i]=='3'||str[i]=='4'||str[i]=='5'||str[i]=='6'||str[i]=='7'||str[i]=='8'||str[i]=='9'||str[i]=='+'||str[i]=='-'){
```

Comment:
A regular expression based test would be better here than comparing like this manually.